src/components/HeaderLoggedIn/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { PureComponent } from 'react'
+import { arrayOf, shape, string } from 'prop-types'
+import cn from 'classnames'
+
+import NavBarMenu from '../NavBarMenu'
+import ClickOutside from '../ClickOutside'
+import logo from '../../assets/logo-wordmark.svg'
+import MobileNavButton from '../MobileNavButton'
+import Badge from '../Badge'
+import defaultAvatar from '../../assets/default-avatar@2x.png'
```

Comment:
We should start organizing the assets folder into subfolders.  Maybe something like:

- global
- header
- footer
- pageName
- componentName
- etc?